modules/log6x-create-log-forwarder.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-create-log-forwarder_{context}"]
+= Creating a log forwarder
+
+The log forwarder provides the following functionality:
+
+* Administrators can control which users are allowed to define log collection and which logs they are allowed to collect.
+* Users who have the required permissions are able to specify additional log collection configurations.
+* Administrators can deploy separate collectors that operate independently according to their collection requirements.
+
+You can create multiple `ClusterLogForwarder` resources using any name and in any namespace.
```

Comment:
For creating a clusterLogForwarder is a:

 - "must" to have before created a serviceAccount in the same namespace that you'll create the clusterLogForwarder and give permissions throught the clusterroles for collecting the desired logs: application, audit, infrastructure similar as it's described in https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/log-forwarding.html#log-collection-rbac-permissions_log-forwarding

@libander